pkg/activator/handler/probe_handler_test.go

```diff
@@ -110,3 +110,29 @@ func mapToHeader(m map[string]string) http.Header {
 	}
 	return h
 }
+
+func BenchmarkProbeHandler(b *testing.B) {
+	baseHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	})
+	handler := ProbeHandler{NextHandler: baseHandler}
+	req := httptest.NewRequest(http.MethodGet, "http://example.com", nil)
+	req.Header.Set(network.ProbeHeaderName, activator.Name)
```

Comment:
Can we do an iteration of 3 different requests here?

1. With probeHeaderName == activator
2. With probeHeaderName != activator
3 With no probeHeaderName set at all

You can check the main activator handler test for how to setup a loop to do different setups. I'd range over  `[]string{"", activator.Name, "bogus"}` in this case probably.